### PR TITLE
Update version_config.cmake and others files for release

### PR DESCRIPTION
**Pull request by paskino**
updates for release 3.5.0

### CODE REVIEW

**Comment by KrisThielemans** [comment]
more CCPi regulariser trouble sadly
```
    from ccpi.filters.regularisers import ROF_TV, FGP_TV, PD_TV, SB_TV, TGV, LLT_ROF, FGP_dTV, NDF, Diff4th
  File "/home/runner/install/python/ccpi/filters/regularisers.py", line 5, in <module>
    from ccpi.filters.cpu_regularisers import TV_ROF_CPU, TV_FGP_CPU, TV_PD_CPU, TV_SB_CPU, dTV_FGP_CPU, TNV_CPU, NDF_CPU, Diff4th_CPU, TGV_CPU, LLT_ROF_CPU, PATCHSEL_CPU, NLTV_CPU
ImportError: /home/runner/install/python/ccpi/filters/cpu_regularisers.cpython-310-x86_64-linux-gnu.so: undefined symbol: _Z14SB_TV_CPU_mainPfS_S_fifiiii
```

Docker builds still stuck due to #826

**Comment by paskino** [comment]
I'm not sure where this bug comes from, it used to work earlier and I am tempted to remove the unit tests for the release and open an issue for next release.

File: SuperBuild/External_CIL.cmake [hunk 1]
```diff
@@ -116,9 +116,10 @@ if(NOT ( DEFINED "USE_SYSTEM_${externalProjName}" AND "${USE_SYSTEM_${externalPr
   add_test(NAME CIL_FRAMEWORK_TESTS_9
            COMMAND ${PYTHON_EXECUTABLE} -m unittest discover -p test_Operator.py
            WORKING_DIRECTORY ${${proj}_SOURCE_DIR}/Wrappers/Python/test)
-  add_test(NAME CIL_FRAMEWORK_TESTS_10
-           COMMAND ${PYTHON_EXECUTABLE} -m unittest discover -p test_PluginsRegularisation.py
-           WORKING_DIRECTORY ${${proj}_SOURCE_DIR}/Wrappers/Python/test)
+  # temporarily disable regularisation toolkit tests due to #832
+  # add_test(NAME CIL_FRAMEWORK_TESTS_10
+  #         COMMAND ${PYTHON_EXECUTABLE} -m unittest discover -p test_PluginsRegularisation.py
+  #         WORKING_DIRECTORY ${${proj}_SOURCE_DIR}/Wrappers/Python/test)
```
Comment (paskino) [inline SuperBuild/External_CIL.cmake:122]:
```suggestion
```

**Comment by paskino** [comment]
@KrisThielemans @evgueni-ovtchinnikov @casperdcl now that the tests are all green (see https://github.com/SyneRBI/SIRF-SuperBuild/pull/829/commits/6e0b7e37b57ddf9269af0defc9afa0dc52b07f78), I suggest to merge this PR and tag the SuperBuild. We should then tackle #832 in a later release.

This PR removes the CCPi Regularisation from the packages built with CIL.

**Review by casperdcl — APPROVED** [review]
lgtm

**Comment by KrisThielemans** [comment]
> This PR removes the CCPi Regularisation from the packages built with CIL.

why? I thought you were just going to disable the tests. If you need to disable the build, then #832 needs to be adjusted.

> I suggest to merge this PR and tag the SuperBuild.

Please merge #830 after this PR. It avoids conflicts for people who have MATLAB installed.

**Comment by paskino** [comment]
The unit test fails because of an import error. I didn't understand why this happens in the GitHub runner. I thought to outright remove the software instead of packaging it with potential import errors.

**Comment by KrisThielemans** [comment]
ok

**Pull request merged**